cli/basic-with-nvm/install_zowe.sh

```diff
@@ -37,4 +38,24 @@ for i in "${plugins[@]}"; do
     fi
 done
 
+if uname -m | grep -q 'arm'; then
+    echo "Unable to install the following plug-ins due to CPU architecture:"
+    for i in "${noarm_plugins[@]}"; do
+        echo $i
+    done
+elif uname -m | grep -q 'aarch'; then
+    echo "Unable to install the following plug-ins due to CPU architecture:"
+    for i in "${noarm_plugins[@]}"; do
+        echo $i
+    done
```

Comment:
Could these be combined into a single `grep` command that matches multiple patterns?
```bash
if uname -m | grep -q 'arm\|aarch'; then
```